### DISCUSSION TITLE
How to specify whether Renovate should update to patch, minor, or major version?

### DISCUSSION THREAD

**Discussion by laurent22** [renovatebot/renovate#18787] [Request Help]
I've recently installed Renovate on my repo and overall it works very well. One thing I've noticed however is that it only upgrades up to patch versions. So for example it will upgrade `glob` from 7.2.0 to 7.2.3, but not to 8.0.3, even though it is available. See here: https://github.com/laurent22/joplin/pull/7085

I assume there's an option to configure this but I couldn't find it. Any information on how to do this?

For information my current config is there, and all our dependencies are pinned: https://github.com/laurent22/joplin/blob/dev/renovate.json

**Comment by viceice** [comment]
probably because of too many rate limited updates?


https://github.com/laurent22/joplin/issues/6996

i think renovate prefer patch over minor over major updates

**Comment by laurent22** [comment]
So does that mean that once it's done with this backlog it will start doing minor and major updates?

**Reply by viceice** [reply]
yes, they are enabled by default. 

you can checkout the logs from app dashboard. (link inside PR footer)
search for package file with updates debug log message.
